Interfaces/IToCall.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Turnip.Classes;
+
+namespace Turnip.Interfaces
```

Comment:
Why do we need these interfaces?

Reply:
This interface implements the behavior when a character calls another character.

Reply:
1. Interface defines, class implements
2. Interfaces are not used to represent objects anywhere.

Generally, we use inheritance for closely related classes: Shape <- Rectangle <- Square, while interfaces allow to make an abstraction over non-related objects, e.g. IRepository <- MySqlRepositury / FileSystemRepository / AzureCosmosRepository.

The point of defining an interface is to use it somewhere. For example, to a SetSomeItemState class constructor we pass the IRepository that allows reading and writing items from any implementation, and allows the business case to not worry about the actual implementation at all.